src/status_im/contexts/wallet/account/bridge_to/view.cljs

```diff
@@ -14,20 +14,20 @@
 (defn- bridge-token-component
   []
   (fn [{:keys [chain-id network-name]} token]
-    (let [network           (rf/sub [:wallet/network-details-by-chain-id chain-id])
-          currency          (rf/sub [:profile/currency])
-          currency-symbol   (rf/sub [:profile/currency-symbol])
-          all-balances      (:balances-per-chain token)
-          balance-for-chain (utils/get-balance-for-chain all-balances chain-id)
-          crypto-formatted  (or (:balance balance-for-chain) "0.00")
```

Comment:
For some reason, this screen was using the `balance` key which we shouldn't. Instead, we should rely on `raw-balance`. So, updated it.